bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/aemform/AEMFormImpl.java

```diff
@@ -278,4 +285,22 @@ public String getHeight() {
         }
         return height;
     }
+
+    private String getFormVersion(ResourceResolver resourceResolver, String formPath) {
```

Comment:
why do need resourceresolver and formpath in args here? it should already be present in the model

Reply:
changed to read from model, removed method arguments.